x-pack/libbeat/processors/add_cloudfoundry_metadata/_meta/fields.yml

```diff
@@ -0,0 +1,39 @@
+- key: cloudfoundry
+  title: Cloud Foundry
+  description: >
+    Cloud Foundry information collected from Cloud Foundry.
+  short_config: false
+  anchor: cloudfoundry-processor
+  fields:
+    - name: cf
```

Comment:
I know this is also part of the previous PR, would it make sense to change `cf` to `cloudfoundry`? I wonder if `cf` could be misleading

Reply:
Yes I agree, especially where metricbeat expects the module name to match the event field name. I will change all of them to `cloudfoundry` which also will match `docker` and `kuberenetes` where they are not abbreviated.